src/middlewared/middlewared/plugins/interface/lag.py

```diff
@@ -76,6 +82,10 @@ def lag_setup(self, lagg, members, disable_capabilities, parent_interfaces, sync
                 self.logger.info('Changing lacpdu_rate on %r to %s', name, info['lacpdu_rate'])
                 iface.lacpdu_rate = info['lacpdu_rate']
 
+            if info['primary_interface'] is not None:
+                self.logger.info('Changing primary interface on %r to %s', name, info['primary_interface'])
+                iface.primary_interface = info['primary_interface']
```

Comment:
Do we need to check if `iface.primary_interface` is already the correct one and in this case do nothing like we do this with `lacpdu_rate`?

Reply:
I'm doing that on lines 43 - 48 up above?